src/detect-mqtt-connack-sessionpresent.c

```diff
@@ -63,7 +63,7 @@ void DetectMQTTConnackSessionPresentRegister (void)
     DetectSetupParseRegexes(PARSE_REGEX, &parse_regex);
 
     DetectAppLayerInspectEngineRegister2("mqtt.connack.session_present", ALPROTO_MQTT,
-            SIG_FLAG_TOSERVER, 1, DetectEngineInspectGenericList, NULL);
+            SIG_FLAG_TOCLIENT, 1, DetectEngineInspectGenericList, NULL);
```

Comment:
@catenacyber Here in the C version of registering this keyword we use a `progress` of 1.  However, over in master when going through Rust its always a `progress` of 0. Should these be 0 over here? What was the reason for always using a progress of 0 instead of carrying over the values from C.

Reply:
I don't know mqtt enough right now. But looks to be an oversight in the Rust registration of keywords?

Reply:
I guess a review of all progresses could be beneficial (some inconsistent progress seen in TLS for multi-buffer IIRC)